packages/office-ui-fabric-react/src/components/MessageBar/MessageBar.scss

```diff
@@ -137,7 +137,6 @@
 
   .ms-MessageBar-dismissal {
     @include focus-border();
-    min-width: 0;
```

Comment:
Is there a reason that you removed this? I'm guessing this was here for a reason, like it was zeroing out some larger min-width set somewhere else.